src/python/test/test_representations.py

```diff
@@ -102,6 +108,10 @@ def test_kernel_distance_consistency():
         f2 = kernelClass.transform(l2)
         f12 = np.array([[kernelClass(l1_, l2_) for l1_ in l1] for l2_ in l2])
         assert f12 == pytest.approx(f2, **tolerance)
+        kernelClass.kernel_approx_ = kernel_approx
```

Comment:
I don't see any `kernel_approx_` in the code, what is this supposed to achieve?

Reply:
It was supposed to be `kernel_approx`!